### ISSUE TITLE
`preferred-alias` rule

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/eslint-plugin-lodash-fp#10]
Enforce the use of a consistent name for each method. For example, `_.pluck` is an alias for `_.map`, so the rule would enforce the use of one over the other.

The rule should by default enforce the use of the non-aliased name, example: `map` instead of `pluck`.
It should take as an option the list of overrides. I'm not sure of the shape of it. Off the top of my head, it'll probably look like this:
```js
"lodash-fp/preferred-alias": ["error", {"aliases": ["pluck", "pipe"]}]
```
In this case, `pluck` should be enforced instead of `map`, and `pipe` instead of `flow`. Any unknown names can be ignored.

Some aliases should not be enforced, like `getOr` and `get`. `getOr` is an alias for `get`, but has a different arity (allowing it to have a default value parameter). I don't know if there are methods where other properties than the fixed arity is different, but they should not be enforced.

If someone wants a stab at it, the current list of fp aliases can be found [here](https://github.com/lodash/lodash/wiki/FP-Guide#aliases)

It might be nice to have presets for aliases, like all the aliases that are there for Ramda users.
`preset: 'ramda'`?
